src/store/group-management/index.ts

```diff
@@ -16,10 +23,16 @@ export enum Stage {
 
 type GroupManagementState = {
   stage: Stage;
+  roomMembers: {
```

Comment:
I'm not sure what this is for. We set it when we add the users to the conversation but we never reference it?

Reply:
Good spot, there is no reason for this state, I think I added this during the earlier stages of developing it and never removed. Thanks!